tests/libtest/lib3026.c

```diff
@@ -61,7 +120,8 @@ int test(char *URL)
   }
 
   for(i = 0; i < tid_count; i++) {
-    int res = pthread_create(&tids[i], NULL, run_thread, &results[i]);
+    results[i] = CURL_LAST;
+    res = pthread_create(&tids[i], NULL, run_thread, &results[i]);
```

Comment:
#nit: Any reason for moving `int res;` a level up?

Reply:
To avoid the error about mixing declarations and code.

Reply:
This should compile fine:
```c
  for(i = 0; i < tid_count; i++) {
    int res;
    results[i] = CURL_LAST;
    res = pthread_create(&tids[i], NULL, run_thread, &results[i]);
```

Reply:
Thanks @vszakats